src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -13,7 +13,6 @@ notSupported:
   - php
   - php.laravel
   - php.symfony
-  - flutter
```

Comment:
do we need to remove `flutter` here? because Dart/Flutter is single-threaded, hence a single hub/scope, or do we need this content available anyway?

Reply:
That would hide stuff like `configureScope` and `withScope` if I understand the include process correctly. However, those two things make sense even in Dart/Flutter, so I would say we need the content available.